src/main/java/net/sf/mzmine/util/maths/similarity/Similarity.java

```diff
@@ -25,7 +25,7 @@
 import net.sf.mzmine.util.maths.Transform;
 
 /**
- * Different similarity measures
+ * Different similarity measures such as COSINE SIMILARITY, PEARSON, SPEARMAN,...

```

Comment:
Similarity class to collect many different similarity measures and some util methods